### PR TITLE
Adding SCS

### PR DESCRIPTION
**Pull request by MatthewCaseres**
The only thing that seems off is that there is no image for SCS, I think I took care of the other stuff?

### CODE REVIEW

File: data/courses.json [hunk 1]
```diff
@@ -163,6 +163,18 @@
...
+    },
+    {
+      "id": "secure-computer-systems",
+      "name": "Secure Computer Systems",
+      "abbrev": "SCS",
+      "code": "CS6238",
+      "semester": "Spring 2021",
+      "author": "Matthew Caseres",
+      "grade": "TBD",
+      "icon": "https://assets.omscs.io/courses/cs6200.png",
```
Comment (m4ttsch) [inline data/courses.json:175]: Ive given you an icon at https://assets.omscs.io/courses/cs6238.png

**Review by m4ttsch — CHANGES_REQUESTED** [review]

File: data/sidebars.json [hunk 2]
```diff
@@ -280,5 +280,14 @@
         "comparing-systems-cont"
       ]
     }
+  ],
+  "secure-computer-systems": [
+    {
+      "section": "Lecture Notes",
+      "notes": [
+        "getting-started",
+        "design-principles"
```
Comment (m4ttsch) [inline data/sidebars.json:289]: You'll need to make sure that these values match the *filenames* of the corresponding lectures. You'll either need to drop the 01/02 from the filenames in `notes/secure-computer-systems` or add them in here.

**Review by m4ttsch — CHANGES_REQUESTED** [review]

File: data/courses.json [hunk 3]
```diff
@@ -163,6 +163,18 @@
...
+      "start": "/secure-computer-systems/getting-started",
```
Comment (m4ttsch) [inline data/courses.json:176]: 01-getting-started

**Review by m4ttsch — CHANGES_REQUESTED** [review]

**Comment by m4ttsch** [comment]
Overall looks pretty good, but I think we may need to drop the numerical prefixes on the filenames for the notes

**Comment by MatthewCaseres** [comment]
I removed the numerical prefixes but is it impossible to do such a thing? The idea is that it gives the files an ordering.

**Review by m4ttsch — APPROVED** [review]
This looks good to go. I think we are ready to merge this.

**Pull request merged**